pkg/daemon/event_recorder.go

```diff
@@ -12,34 +12,33 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/log"
 
 	snclientset "github.com/k8snetworkplumbingwg/sriov-network-operator/pkg/client/clientset/versioned"
+	"github.com/k8snetworkplumbingwg/sriov-network-operator/pkg/global/vars"
 )
 
 type EventRecorder struct {
 	client           snclientset.Interface
-	node             string
 	eventRecorder    record.EventRecorder
 	eventBroadcaster record.EventBroadcaster
 }
 
 // NewEventRecorder Create a new EventRecorder
-func NewEventRecorder(c snclientset.Interface, n string, kubeclient kubernetes.Interface) *EventRecorder {
+func NewEventRecorder(c snclientset.Interface, kubeclient kubernetes.Interface) *EventRecorder {
 	eventBroadcaster := record.NewBroadcaster()
 	eventBroadcaster.StartStructuredLogging(4)
 	eventBroadcaster.StartRecordingToSink(&typedv1core.EventSinkImpl{Interface: kubeclient.CoreV1().Events("")})
 	eventRecorder := eventBroadcaster.NewRecorder(scheme.Scheme, corev1.EventSource{Component: "config-daemon"})
 	return &EventRecorder{
 		client:           c,
-		node:             n,
 		eventRecorder:    eventRecorder,
 		eventBroadcaster: eventBroadcaster,
 	}
 }
 
 // SendEvent Send an Event on the NodeState object
 func (e *EventRecorder) SendEvent(eventType string, msg string) {
-	nodeState, err := e.client.SriovnetworkV1().SriovNetworkNodeStates(namespace).Get(context.Background(), e.node, metav1.GetOptions{})
+	nodeState, err := e.client.SriovnetworkV1().SriovNetworkNodeStates(namespace).Get(context.Background(), vars.NodeName, metav1.GetOptions{})
```

Comment:
do we really want this ? or should each component that cares about the node name have it as a struct member ?
that is its New*() method to still accept nodeName  as parameter. the initializer would then pass in vars.NodeName

Reply:
I don't need we need to duplicate data there is no point of that if you ask me.
everything that is in the vars folder is a long-live configuration that is specific for the all run of the daemon or the operator.

it's not possible to have one node name in one package/struct and a different one in another that is why I wanted to have the vars folder to be global for all application

Reply:
ack, lets go with this approach.

Reply:
I really don't like this approach with global variables but since team agrees on it, I won't block PR for this